src/Services/Azure/Authentication/CustomChainedCredential.cs

```diff
@@ -61,9 +61,25 @@ private static TokenCredential CreateCredential(string? tenantId)
             return CreateBrowserCredential(tenantId, authRecord);
         }
 
-        return new ChainedTokenCredential(
-            CreateDefaultCredential(tenantId),
-            CreateBrowserCredential(tenantId, authRecord));
+        var creds = new List<TokenCredential>();
+        var vsCodeCred = CreateVsCodeBrokerCredential(tenantId);
+        bool isVsCodeContext = !string.IsNullOrEmpty(Environment.GetEnvironmentVariable("VSCODE_PID"));
```

Comment:
This is worth a source code comment on the env variable

Reply:
updated.